### PR TITLE
one new file created by pranita

### PR DESCRIPTION
**Pull request by pranitamandal**
review the code

### CODE REVIEW

File: LogoutDummy/src/test/java/Logout.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+public class Logout {
+	public static void main(String[] args) {
+		String name="Logout";
+		System.out.println("Logout is done");
+	}
```
Comment (Jaikumar1998) [inline LogoutDummy/src/test/java/Logout.java:6]: dont use this code

**Review by Jaikumar1998 — COMMENTED** [review]
ok done project is over

File: LogoutDummy/src/test/java/Logout.java (hunk 1 above)
Comment (pranitamandal) [reply LogoutDummy/src/test/java/Logout.java:6]: fixed

**Comment by Jaikumar1998** [comment]
merged

**Pull request merged**